aim/web/ui/src/pages/RunDetail/RunDetail.tsx

```diff
@@ -62,7 +62,13 @@ function RunDetail(): React.FunctionComponentElement<React.ReactNode> {
 
   // TODO: add code splitting(lazy loading)
   const tabContent: { [key: string]: JSX.Element } = {
-    overview: <RunOverviewTab runData={runData} runHash={runHash} />,
+    overview: (
+      <RunOverviewTab
+        runHash={runHash}
+        runData={runData}
+        runParams={runData?.runParams}
```

Comment:
let's remove `runParams` props and use it from `runData` as we anyway pass the `runData `